README.md

```diff
@@ -57,6 +73,15 @@ This task requires that Docker is installed on the Azure Pipelines agent to push
 Registry. For more information on how to install Docker on the agent, please see
 [this document](https://docs.docker.com/get-docker/).
 
+In addition, users running this task with a Windows agent may encounter an issue with not being able to pull down
```

Comment:
`with a Windows agent may encounter an issue` => Are there some cases where it works? From the other paragraph above it felt like a hard limitation, not something that might depend on other stuff. Could you please clarify?

Reply:
OK now I got it. Please consider adding a complement to this sentence: `to pull down the Oryx builder to create runnable application images from provided application source. Please see a workaround below if you prefer to use Windows agents.` or something of that kind?

Reply:
That's a good idea -- let me rephrase what I typed above in the other comment about the scenarios that should work for users so they understand what does and doesn't work.

Reply:
Thanks!